core/app/models/spree/payment/processing.rb

```diff
@@ -35,12 +35,37 @@ def process!
       end
 
       def authorize!
-        handle_payment_preconditions { process_authorization }
```

Comment:
- https://github.com/search?q=org%3Asolidusio+handle_payment_preconditions&type=code
- https://github.com/search?q=org%3Asolidusio-contrib+handle_payment_preconditions&type=code

Not used in any extension 🎉